.github/workflows/release-docker.yml

```diff
@@ -2,7 +2,8 @@ name: Build and Push Docker Image
 
 on:
   release:
-    types: [created]
+    types: [published]
```

Comment:
The v0.6.0 release must have had an automatic draft created that doged the `created` type.  Hopefully this helps.